llm_toolkit/prompt_builder.py

```diff
@@ -751,6 +752,29 @@ def build_triager_prompt(self, benchmark: Benchmark, driver_code: str,
     # Do nothing for jvm project now.
     return self._prompt
 
+  def post_process_generated_code(self, generated_code: str) -> str:
+    """Allows prompt builder to adjust the generated code."""
+    # For unknown reasons, the LLM model keeps using wrong
+    # FuzzedDataProvider::consumeObject() or FuzzedDataProvider::getObject()
+    # methods to generate random java.lang.Object instance. These methods are
+    # not valid in FuzzedDataProvider. The fixes here change the calling of
+    # them to FuzzedDataProvider::consumeString(int) instead.
+    generated_code = generated_code.replace(
+        'data.consumeObject()', 'data.consumeString(data.remainingBytes()/2)')
+    generated_code = generated_code.replace(
+        'data.getObject()', 'data.consumeString(data.remainingBytes()/2)')
```

Comment:
Could we have some samples of this substitution in the PR description? Why are `data.consumeString` the right substition?